Dockerfile

```diff
@@ -8,12 +8,13 @@ WORKDIR /RackHD/on-http
 RUN mkdir -p ./node_modules \
   && ln -s /RackHD/on-core ./node_modules/on-core \
   && ln -s /RackHD/on-core/node_modules/di ./node_modules/di \
-  && npm install --ignore-scripts --production \
```

Comment:
@benbp , line 17 actually removed the dev dependancies which will reduce the filesize of the image. Does that meets your expectation? 

I remove ```--production``` from line 11 because some of the dev dependances are needed by genaration task doc. After taskdoc finish, I remove the dev dependance in line 7.